src/Chainweb/Chainweb.hs

```diff
@@ -812,14 +810,15 @@ withChainwebInternal conf logger peer rocksDb pactDbDir resetDb inner = do
         syncOne :: (BlockHeader, ChainResources logger) -> IO ()
         syncOne (bh, cr) = do
             let pact = _chainResPact cr
-            let logCr = logFunctionText $ _chainResLogger cr
+            let logCr = logFunctionText
+                    $ addLabel ("component", "pact")
+                    $ addLabel ("sub-component", "init")
+                    $ _chainResLogger cr
             let hsh = _blockHash bh
             let h = _blockHeight bh
             logCr Info $ "pact db synchronizing to block "
                 <> T.pack (show (h, hsh))
-            payload <- payloadWithOutputsToPayloadData
-                <$> casLookupM payloadDb (_blockPayloadHash bh)
-            void $ _pactValidateBlock pact bh payload
+            void $ _pactSyncToBlock pact bh
```

Comment:
Did you consider simply adding a flag to `_pactValidateBlock` to do incremental replay? Adding a queue RPC is expensive when this seems to be essentially a piece of advice on how to replay.

Reply:
I did consider it. But the use case is very different. Using pactValidateBlock for synchronizing the pact db with the consensus db always felt a bit like a misuse of it.

An `incrementalRewindTo` flag on `pactValidateBlock` would be feel a bit far-fetched and would only really be used for the initial db synchronization. So, adding an new API request for the synchronization seemed more natural to me.

Reply:
> Using pactValidateBlock for synchronizing the pact db with the consensus db always felt a bit like a misuse of it.
👍